### DISCUSSION TITLE
DataTable: Missing slotsProps to paginatorstart and paginatorend

### DISCUSSION THREAD

**Discussion by jinyongp** [primefaces/community#2527] [PrimeVue]
https://github.com/primefaces/primevue/issues/4907#issue-2018892328

> On the documentation page for the PrimeVue DataTable Paginator Template [here](https://primevue.org/datatable/#paginator_template), it is observed that the `paginatorstart` and `paginatorend` slots are mentioned, but there seems to be a lack of slot props that would allow users to customize the display of the paginator, similar to the original Paginator component [here](https://primevue.org/paginator/#customcontent).
> 
> [codesandbox.io/p/devbox/eloquent-montalcini-9wzvn4?file=/src/App.vue:42,42](https://codesandbox.io/p/devbox/eloquent-montalcini-9wzvn4?file=%2Fsrc%2FApp.vue%3A42%2C42)

The issue still hasn't been fixed in primevue v4.0.4.

**Comment by kozubov** [comment]
Have the same problem.
Can't output CurrentPageReport in start slot

`<DataTable :value="trips" dataKey="id" removableSort :paginator="true"
                       :rows="10"
                       paginatorTemplate="RowsPerPageDropdown FirstPageLink PrevPageLink PageLinks NextPageLink LastPageLink "
                       :rowsPerPageOptions="[5, 10, 20]"
                       :totalRecords="120">`
                `<template #paginatorstart="{ first, last, totalRecords }">`
                    `<span >Showing {{ first }} to {{ last }} of {{ totalRecords }}</span>`
               `</template>`
             Did you solve this problem or find solution?
             
![image](https://github.com/user-attachments/assets/24e90a86-a83b-44aa-b0ff-883ec1b63f9f)
